src/command.ts

```diff
@@ -83,6 +85,9 @@ namespace Command
 			case Command.ExitStatus.BadInvocation:
 				message.reply("Bad invocation. From the documentation: \n\n" + command.Documentation.Invocation);
 				break;
+			case Command.ExitStatus.NotInVoiceChannel:
+				message.reply("You must be in the voice channel to vote in this vote");
```

Comment:
`vote in this vote` sounds a bit awkward, maybe use `poll` for the second `vote`?

Reply:
One issue with this, is that it does mean anyone can join the voice channel just to vote, but there isn't really a way to go against that anyway

Reply:
This has been one of my biggest problems with the bot. I've been thinking of ways to try to prevent users from doing this, but I haven't tried to implement them yet. Maybe I'll start working on this soon.

Reply:
One way would be to limit it to people who were in it on creation (making a variable of who was in it when the poll was created).